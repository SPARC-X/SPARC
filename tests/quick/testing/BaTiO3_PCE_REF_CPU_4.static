***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Ba:
      0.9882651394       0.9500437390       0.9854007162
Fractional coordinates of Ti:
      0.4883051330       0.5082999951       0.4491430331
Fractional coordinates of O:
      0.5311585887       0.9423479712       0.5054895152
      0.9293054969       0.4556304936       0.4885723786
      0.4720456043       0.4781589886       0.0261939929
Total free energy (Ha): -1.318853332804203E+02
Atomic forces (Ha/Bohr):
  3.8256162193E-03  -6.7453319238E-02  -1.7763915305E-01
  1.0277796032E-01   1.2274504225E-01  -2.7959857950E-01
 -6.0578794493E-02   2.4597261436E-01  -4.2929973771E-01
 -1.9813652770E-01  -6.3959233238E-01   5.3915212237E-01
  1.5211174565E-01   3.3832799502E-01   3.4738534789E-01
