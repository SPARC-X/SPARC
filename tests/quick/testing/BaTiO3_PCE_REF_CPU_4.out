***************************************************************************
*                       SPARC (version Oct 21, 2020)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Fri Mar 19 00:54:29 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.63 7.63 7.63 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Gaussian
SMEARING: 0.007349864435
CHEB_DEGREE: 17
NSTATES: 29
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-05
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-05
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.59E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: BaTiO3
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 2
NP_DOMAIN_PARAL: 1 1 2
NP_DOMAIN_PHI_PARAL: 1 2 2
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  4
Mesh spacing                       :  0.508667 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  BaTiO3.out_51
Total number of atom types         :  3
Total number of atoms              :  5
Total number of electrons          :  40
Atom type 1  (valence electrons)   :  Ba 10
Pseudopotential                    :  Ba.psp8
Pseudocharge radii of atom type 1  :  6.61 6.61 6.61 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  Ti 12
Pseudopotential                    :  Ti.psp8
Pseudocharge radii of atom type 2  :  6.10 6.10 6.10 (x, y, z dir)
Number of atoms of type 2          :  1
Atom type 3  (valence electrons)   :  O 6
Pseudopotential                    :  O.psp8
Pseudocharge radii of atom type 3  :  6.61 6.61 6.61 (x, y, z dir)
Number of atoms of type 3          :  3
Estimated total memory usage       :  5.20 MB
Estimated memory per processor     :  1.30 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.6459727275E+01        1.982E-01        5.516
2            -2.6386112248E+01        6.055E-02        1.411
3            -2.6380284176E+01        4.160E-02        1.363
4            -2.6379652648E+01        3.909E-02        1.342
5            -2.6377398167E+01        1.425E-02        1.055
6            -2.6377083907E+01        3.541E-03        1.500
7            -2.6377074330E+01        2.673E-03        1.353
8            -2.6377069268E+01        1.428E-03        1.411
9            -2.6377066984E+01        6.543E-04        1.184
10           -2.6377066590E+01        9.120E-05        1.372
11           -2.6377066817E+01        6.669E-05        1.513
12           -2.6377066392E+01        1.151E-05        1.330
13           -2.6377066781E+01        4.779E-06        1.132
Total number of SCF: 13    

k = [0.000, 0.000, 0.000]
Occupation of state 26 = 0.000000000000000.
Occupation of state 29 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.6377066656E+01 (Ha/atom)
Total free energy                  : -1.3188533328E+02 (Ha)
Band structure energy              : -1.1200043727E+01 (Ha)
Exchange correlation energy        : -2.1485252114E+01 (Ha)
Self and correction energy         : -2.0340676450E+02 (Ha)
Entropy*kb*T                       : -1.1232750201E-08 (Ha)
Fermi level                        :  2.8846431696E-01 (Ha)
RMS force                          :  4.7571718051E-01 (Ha/Bohr)
Maximum force                      :  8.5966356576E-01 (Ha/Bohr)
Time for force calculation         :  0.033 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  21.646 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
